src/test/java/studyconnect/StudyConnectApplicationTest.java

```diff
@@ -0,0 +1,17 @@
+package studyconnect;
+
+import com.studyconnect.StudyConnectApplication;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.AvoidStarImportCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Using the '.*' form of import should be avoided - org.junit.jupiter.api.Assertions.*.